client/src/App.js

```diff
@@ -102,7 +102,7 @@ class App extends Component {
             <Clients />
           </Route>
           <Route exact path="/settings">
-            <Settings />
+            <Settings color={this.state.data.color} size={this.state.data.size} />
```

Comment:
instead of passing the state here, we could use the `useState` hook to pass the `setState` function for each setting directly. https://reactjs.org/docs/hooks-reference.html#usestate

Reply:
@hannahmkim so the way I set it up is that `useState` hook is being set in the parent component `<App>`. When it updates state, it passes properties down to the child component `<Settings>`. Whenever state updates, the child component will update automatically because of the props being updates at the parent level via the `updateServer` method -> `client.onMessage` method flow.

Let me know if that makes sense.